src/styles/blocks/footer.scss

```
@@ -0,0 +1,183 @@
+.footer {
+  width: 100%;
+  background-color: $header-bg;
+  border-radius: $header-border-radius $header-border-radius 0 0;
+  margin-top: -25px;
```

```suggestion
  margin-top: -25px;
```
Your `footer` is BEM-block. You cannot use margins for blocks